GitUI/BranchTreePanel/RepoObjectsTree.ContextActions.cs

```diff
@@ -151,5 +167,26 @@ private void contextMenu_Opening(object sender, CancelEventArgs e)
             ContextMenuBranchSpecific(contextMenu);
             ContextMenuRemoteRepoSpecific(contextMenu);
         }
+
+        /// <inheritdoc />
+        public TMenuItem CreateMenuItem<TMenuItem, TNode>(Action<TNode> onClick, string captionKey, string toolTipKey, Bitmap icon = null)
+            where TMenuItem : ToolStripItem, new()
+            where TNode : class, INode
+        {
+            var result = new TMenuItem();
+            result.Image = icon;
+            result.Text = new TranslationString(captionKey).Text;
```

Comment:
I don't believe this is a correct way of using `TranslationString`, as I am pretty sure they won't be picked up by a translation app and, as a result, won't be emitted into xlf files bound for translations.
Nor that they would get reloaded with a different language.

Please follow the established patterns for the localisation.

Reply:
I'm still wrapping my head on how TranslationString is used [loading...]

Reply:
I'll close this conversation, as there's another above regarding translations